app/uk/gov/hmrc/taxenrolmentassignmentfrontend/logging/LoggingEvent.scala

```diff
@@ -85,11 +85,22 @@ object LoggingEvent {
       )
     )
 
-  def logES2ErrorFromEACD(
-                           credId: String,
-                           statusReturned: Int,
-                           eacdErrorMsg: String = "N/A"
-                         ): LoggingEvent =
+  def logUnexpectedResponseFromUsersGroupSearch(
+    credId: String,
+    statusReturned: Int
+  ): LoggingEvent =
+    Error(
+      Event(
+        "[UsersGroupSearchConnector][getUsersDetails]",
+        errorDetails = Some(
+          s"Users Group Search return status of $statusReturned for credID $credId"
```

Comment:
Can we add the error message to this as well?

Reply:
added